lib/AnyEvent/WebSocket/Client.pm

```diff
@@ -56,14 +56,18 @@ use PerlX::Maybe qw( maybe provided );
    # close the connection (either inside or
    # outside another callback)
    $connection->close;
- 
+
  });
 
  ## uncomment to enter the event loop before exiting.
  ## Note that calling recv on a condition variable before
  ## it has been triggered does not work on all event loops
  #AnyEvent->condvar->recv;
 
+# fixed bad bad pod encoding
+
+=encoding utf8
```

Comment:
dzil adds this so it isn't really necessary.

Reply:
I usually pod test.. So Hard errors freak me out a bit.

Reply:
pod testing happens after dzil builds the dist:

```
xt/author/pod.t .................. ok   
xt/author/pod_coverage.t ......... ok   
xt/author/pod_spelling_common.t .. ok   
xt/author/pod_spelling_system.t .. ok   
```

It's sort of pointless to test the incomplete POD.